packages/services/examples/typescript-browser-with-output/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "private": true,
+  "name": "typescript-browser-with-output",
+  "dependencies": {
+    "@jupyterlab/outputarea": "^0.8.2",
+    "@jupyterlab/rendermime": "^0.8.2",
+    "@jupyterlab/services": "0.46.0",
+    "es6-promise": "^4.1.0"
+  },
+  "scripts": {
+    "build": "tsc && webpack",
+    "clean": "rimraf build"
+  },
+  "devDependencies": {
+    "rimraf": "^2.5.2",
+    "tslint": "~4.5.0",
+    "typescript": "~2.2.0",
```

Comment:
I believe cause of the error you were running into is here. We updated to TS 2.4.1 a day or two ago, and having and older version causes some type errors.